main.go

```diff
@@ -51,11 +56,10 @@ func main() {
 	version := flagSet.Bool("v", false, "Display version")
 	err := flagSet.Parse(os.Args[1:])
 	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+		logger.Fatalln(err)
 	}
 	if *version {
-		fmt.Println(VERSION)
+		logger.Infoln(VERSION)
```

Comment:
What does this output look like? Is there a datestamp or anything (there shouldn't be)